src/screens/Portal/index.tsx

```diff
@@ -28,14 +30,29 @@ export function Portal() {
   }
 
   return (
-    <SimpleSheet backgroundColor="white" scrollEnabled={false}>
-      <Box paddingVertical="44px" paddingHorizontal="32px" height="full" background="surfaceSecondary">
+    <Box style={styles.container}>
+      <TapToDismiss />
+      <Panel height={params.sheetHeight} innerBorderWidth={0} outerBorderWidth={2.5} style={styles.panel}>
```

Comment:
Is this portal always used in sheets that are not inset from the bottom safe area? If so, would set the `outerBorderWidth` to 0, otherwise in dark mode you get the border width on the sides of the sheet that don't properly flow to the bottom 

<img width="769" alt="Screenshot 2025-04-21 at 11 06 52 AM" src="https://github.com/user-attachments/assets/e941f4d7-1ce8-4385-81d7-1b62d0cf1a9e" />